custom_components/ha-glocaltokens/api.py

```diff
@@ -85,15 +81,14 @@ def get_alarms_and_timers_from(self, device, endpoint):
 
     def get_google_devices_information(self):
         _LOGGER.debug("Fetching data...")
-        devices = self._client.get_google_devices_json()
+        devices = self._client.get_google_devices()
 
         for device in devices:
             # To avoid keyerror's
             timers = []
             alarms = []
 
-            device[DEVICE_IP] = "192.168.0.205"  # For testing purpose only
-            device[DEVICE_PORT] = PORT  # For testing purpose only
+            device.ip = "192.168.0.205"
```

Comment:
Please remove your hardcoded ip and port from `device`. 
If autodiscovery doesn't work for you yet - you can just locally hardcode the IP for testing purposes. Having it generic in this PR will allow others to work on the same branch 👍

Reply:
Sorry, I forgot that. Will remove it right away!